### DISCUSSION TITLE
Add support of madMAx43v3r/chia-plotter

### DISCUSSION THREAD

**Discussion by fabdu117** [ericaltendorf/plotman#786] [Ideas]
I would like to propose the support of https://github.com/madMAx43v3r/chia-plotter.

Chia-plotter is very faster than official plotter, because it uses all the processor core.

**Comment by racergoodwin** [comment]
This is already being talked about, though since there is talk of it being merged into the Chia network it isn't a high priority, yet.
For now, you could set up plotman to use the archiving feature alongside madMAx43v3r's plotter. Edit the config.yaml and change the autostart_plotting: True to False

**Reply by fabdu117** [reply]
Thank you is very useful.

**Reply by jayhohoho2019** [reply]
Or "plotman archive" will do it seems.
